cronjob/bro-pkg-web-updater.php

```diff
@@ -50,18 +59,18 @@ function required_exec($cmd) {
 
 $pkgcount = count($pkgarray);
 if ($pkgcount > 0) {
-    echo "Processing $pkgcount packages";
+    echo "Processing $pkgcount packages\n\n";
 }
 foreach ($pkgarray as $pkg) {
-    echo ".";
+    echo "Processing $pkg\n";
 
     // Remove trailing 'local package' information
     $pkg = preg_replace('/\s.*$/', '', $pkg);
 
     // Get package info for the current package
     $zkg_output = null;
     $zkg_res = null;
-    exec("$zkg_exec info $pkg --json --nolocal --allvers", $zkg_output, $zkg_res);
+    timeout_exec("$zkg_exec info $pkg --json --nolocal --allvers", $zkg_output, $zkg_res);
```

Comment:
This call to `zkg` is creating a copy of every package in /root/.zkg/scratch, which eventually fills up the disk on a t2.micro AWS instance and blocks other packages from being read. It should probably remove that temp directory for each package once it's done scanning.

Reply:
Fixed in https://github.com/zeek/zeek-pkg-web/pull/7/commits/8f8fc997e02a632b4d3d1fe999df988513a0b2d1